### ISSUE TITLE
Share tool link is broken

### ISSUE THREAD

**Issue by hexylena** [galaxyproject/galaxy#937]
After upgrading to dev, accessing a shared tool link, e.g. https://fqdn/galaxy/root?tool_id=cat1 returns the message "Uncaught error."

This seems to be because somewhere in the JS the URL is build to access the API for the toolform, and the url that gets built is `https://fqdn/galaxy/api/tools/cat1/build?&`. Removing the ampersand results in the API responding to the request, and everything working.

Tagged ui/ux because it's the JS that's building the URL.

**Comment by martenson** [comment]
https://test.galaxyproject.org is running yesterday's dev and does not have this problem so I assume proxy prefix again
ping @guerler

**Comment by hexylena** [comment]
@guerler, related, today I noticed this URL being requested when I loaded a tool from the toolbox:

```
https://fqdn/galaxy/api/tools/edu.tamu.cpt.fasta.codonopt/build?tool_version=0.1&&
```

**Comment by guerler** [comment]
This has been fixed right?

**Comment by hexylena** [comment]
not as far as I know

**Comment by martenson** [comment]
Why was this issue closed @guerler ?

**Comment by guerler** [comment]
ping @erasche

**Comment by hexylena** [comment]
@martenson @guerler and I chatted about it on IRC. I tested and it was fixed in :dev. It seems to have been fixed recently, I assume it was in #947.

**Comment by martenson** [comment]
Cool. Thank you @erasche and @guerler
